src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -0,0 +1,698 @@
+============== file: test/test.js ==============
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var assert = require('assert');
+var service = require('../src/').v1();
+
+describe('LibraryServiceClient', function() {
+  describe('createShelf', function() {
+    it('Invokes createShelf without error', function(done) {
```

Comment:
Please start the message with lower letter (`'invokes ...`).

Mocha concatenates the message, so in this context, the actual output message would look like
```
LibraryServiceClient createShelf invokes createShelf without error
```
Starting lowercase letter is preferred to make this look like a normal sentence.